client/coral-framework/helpers/plugins.js

```diff
@@ -11,7 +11,7 @@ import {loadTranslations} from 'coral-framework/services/i18n';
 export const pluginReducers = merge(
   ...plugins
     .filter((o) => o.module.reducer)
-    .map((o) => ({...o.module.reducer}))
+    .map((o) => ({[o.module.reducer.name] : o.module.reducer}))
```

Comment:
You can always nest reducers. We don't want to run into the situation were plugins accidentally overwrites another reducer.